src/servers/src/http.rs

```diff
@@ -525,11 +536,18 @@ impl HttpServer {
 
         if let Some(prom_handler) = self.prom_handler.clone() {
             router = router.nest(
-                &format!("/{HTTP_API_VERSION}/prometheus"),
+                &format!("/{HTTP_API_VERSION}/prom_store"),
```

Comment:
We'd better not to change this because some users may have been using this api for remote write. Could you please test if we can keep `/prometheus` here?

Reply:
sure, if we mean to keep `/prometheus`, then i shall merge these two routes

Reply:
Yes. We already have `/prometheus` for both API and remtoe read/write on Cloud. It would be nice to keep this consistent.

Reply:
fixed